### DISCUSSION TITLE
Regarding those qr codes

### DISCUSSION THREAD

**Discussion by adiibytes** [Cyfrin/foundry-full-course-cu#1571] [General]
Hello i have just started the course yesterday and completed lesson 1. I have got qr codes in the youtube and when i scan it and go to the page i am not understanding anything how to do challenges and get that NFT. Can anyone guide me how to go along with that?

**Comment by EngrPips** [comment]
Hello @adiibytes , please show to me where the QR code took you to when you scan it.

**Reply by adiibytes** [reply]
![Screenshot (260)](https://github.com/Cyfrin/foundry-full-course-f23/assets/157119800/51a1f8ab-a104-471b-9476-fe43535ccf3b)

**Reply by EngrPips** [reply]
Awesome, now click on the `Write Contract` tab and you should see the functions you can call on the contract , one of those function is name `solveChallenge` or so , you should read what input the function is expecting and input them to the function as argument and when you call that function on the contract it should in return mint you the Nft for the section [provided you call the function correctly] , Please let me know if you fins anything confusing. 

Looking forward to hear back from you.

**Reply by adiibytes** [reply]
<img width="255" alt="image" src="https://github.com/Cyfrin/foundry-full-course-f23/assets/157119800/2957c36c-ecbd-4749-9527-4e71a0556807">
This one??

**Reply by EngrPips** [reply]
I would assume you called the right transaction

**Comment by Pedrog75** [comment]
@EngrPips > thank for your answer.
How did you know that we should go to the write contract tab instead of copy/pasting the code to remix and getting crazy 😅😅...

**Reply by EngrPips** [reply]
Anytime, friend, I have seen Patrick go to the write contract tab in a video.
